src/changelogs/fetch_changelogs.py

```diff
@@ -0,0 +1,69 @@
+import os
+from os import listdir
+from os.path import isfile, join
+import feedparser
+from bs4 import BeautifulSoup
+from urllib import request
+
+
+class ChangelogFetcher:
+    def __init__(self):
+        self.changelogs_by_date = {}
+
+    def get_rss(self, rss_url, update_Existing = False):
```

Comment:
Should `update_Existing` have a capital "E"?

Reply:
fixed!